interpreter/cling/lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -207,7 +207,8 @@ class Azog: public RTDyldMemoryManager {
 
   void deregisterEHFrames() override {
 #ifdef LLVM_ON_WIN32
-    platform::DeRegisterEHFrames(Addr, Size);
+    // platform::DeRegisterEHFrames(Addr, Size);
```

Comment:
I'm not sure what this is doing.
If exception handling on X64 broke because of the LLVM upgrade it should be fixed.
If exception handling on X86 still isn't implemented, disabling for X64 doesn't seem like a solution.